<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	border: 1px;
	border-bottom-color: rgb(150, 150, 150);
}
QLabel{
	font: 75 10pt &quot;Arial&quot;;
	color: rgba(100, 100 , 100, 255);
	background-color: rgb(255, 255, 255);
	border-radius: 50pt;
}

QLineEdit{
	background-color: rgb(255, 255, 255);
	border: None;
	
	color: rgb(121, 121, 121);
}

QPushButton{
	font: 75 10pt &quot;Arial&quot; bold;
	color: rgb(255, 255, 255);
	border: 1px solid rgb(30, 210, 80);
	border-radius: 10px;
	background-color: rgb(30, 210, 80);
}

QComboBox{
	font: 75 10pt &quot;Arial&quot; ;
	background-color: rgb(255, 255, 255);
	border: None;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="viewerWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="buttons">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="izoViewBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Izometric View</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/Icons/Icons/tv.svg</normaloff>:/Icons/Icons/tv.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="snapshootBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Take snapshot</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/Icons/Icons/camera.svg</normaloff>:/Icons/Icons/camera.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fullscreenBtn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Fullscreen</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/Icons/Icons/maximize-2.svg</normaloff>:/Icons/Icons/maximize-2.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="screenshootBtns">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="retryBtn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Retry</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/Icons/Icons/chevrons-left.svg</normaloff>:/Icons/Icons/chevrons-left.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
